source/Halibut/Util/RetryPolicy.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Halibut.Util
 {
-    class RetryPolicy
+    public class RetryPolicy
```

Comment:
Is forced to be public, but you can't actually set it on the runtime. So I guess someone could use this class but not with halibut.